documentation/src/main/asciidoc/spatial.asciidoc

```diff
@@ -367,13 +362,10 @@ repetition is needed to allow you to define Queries with any tool.
 [NOTE]
 .Sorting and null values
 ====
-Using distance sort on non `@Spatial` enabled entities and/or with a non spatial Query will have also
-unexpected results as entities non spatially indexed and/or with null values for latitude or
-longitude will be considered to be at (0,0)/(lat,0)/(0,long)
-
-Using distance sort with a spatial query on spatially indexed entities having, potentially, `null`
-values for latitude and/or longitude is safe as they will not be found by the spatial query and so
-won't be sorted
+When a spatial field on an entity has a `null` value for either its latitude or longitude (or both),
+the resulting distance will always be `Double.MAX_VALUE` (`1.7976931348623157e+308`),
+and the entity will be ranked accordingly:
+last if the sort is ascending, or first if the sort is descending.
```

Comment:
Two things:

 1. The fact that a value is listed or not has nothing to do with sorts. It's related to distance constraints (distance queries), which are orthogonal to the sorts.
 2. *If* you also have a constraint on distance, the entities with null latitude/longitude will probably not appear in the search results, since `Double.MAX_VALUE` is larger than any max distance you might have set (except ``Double.MAX_VALUE`).

I'll update the doc to make that more obvious.